### PR TITLE
fix: added v2 support and fixed deep link 

### PR DESCRIPTION
**Pull request by ami-aman**
closes: https://github.com/customerio/issues/issues/8923

Add iOS v2 support in ami app and fix deep link issue

### CODE REVIEW

File: ios/Notification Service/NotificationService.swift [hunk 1]
```diff
@@ -1,13 +1,18 @@
...
 import CioMessagingPush
+import CioTracking
+
 class NotificationService: UNNotificationServiceExtension {
    
       override func didReceive(_ request: UNNotificationRequest,
                                withContentHandler contentHandler:
                                @escaping (UNNotificationContent) -> Void) {
         
+        CustomerIO.initialize(siteId: "94541e1bbff594682089", apiKey: "914ea21ebfa87bf771f4", region: .US) { config in
```
Comment (mrehan27) [inline ios/Notification Service/NotificationService.swift:11]: Shouldn't we keep keys in ignored files and use from there? 👀

Comment (ami-aman) [reply ios/Notification Service/NotificationService.swift:11]: Oops! My bad 🤣 
Thanks for noticing this !

**Review by mrehan27 — APPROVED** [review]

File: ios/Notification Service/NotificationService.swift (hunk 1 above)
Comment (ami-aman) [reply ios/Notification Service/NotificationService.swift]: This doesn't look like a quick change. I will have to update CI and secret keys. Will have to open another PR to fix this. I have removed the keys and adding a note in Todo marker.

**Review by xtreem88 — APPROVED** [review]

**Comment by Shahroz16** [comment]
Shouldn't we just bump to **2.0.1**? As it will contain changes from both?

**Comment by ami-aman** [comment]
> 

Changes from both ? 🤔

**Comment by Shahroz16** [comment]
> > 
> 
> Changes from both ? 🤔

Changes from both released 😄

**Comment by matt-frizzell** [comment]
@ami-aman I think (but I'm not sure) that @Shahroz16 is saying you should use the 2.0.1 iOS SDK version instead of the 2.0.0 version as it includes the major changes in 2.0.0 and the hotfix that was added after that initial release. No reason to just grab some of the changes.

**Comment by ami-aman** [comment]
Thanks, Shahroz and Matt for suggesting the change. I have bumped the version to 2.0.1 in the latest commit.

**Pull request merged**